Packs/RiskIQDigitalFootprint/IncidentFields/incidentfield-RiskIQ_Asset_Okta_Zone_ID.json

```diff
@@ -0,0 +1,38 @@
+{
+    "id": "incident_riskiqassetoktazoneid",
+    "version": -1,
+    "modified": "2020-07-15T12:02:11.549485407+05:30",
+    "name": "RiskIQ Asset Okta Zone ID",
+    "ownerOnly": false,
+    "placeholder": "Okta Zone ID for which the RiskIQAsset of the IP Address type is to be whitelisted.",
+    "description": "Okta Zone ID for which the RiskIQAsset of the IP Address type is to be whitelisted.",
+    "cliName": "riskiqassetoktazoneid",
+    "type": "shortText",
+    "closeForm": false,
+    "editForm": true,
+    "required": false,
+    "script": "",
+    "fieldCalcScript": "",
+    "neverSetAsRequired": false,
+    "isReadOnly": false,
+    "selectValues": null,
+    "validationRegex": "",
+    "useAsKpi": false,
+    "locked": false,
+    "system": false,
+    "content": true,
+    "group": 0,
+    "hidden": false,
+    "associatedTypes": null,
```

Comment:
We did not associate it with any incident type since this will be used for a playbook that is triggered for indicators that will be executed using a job that can have any type of incident.